vertx-web-graphql/src/main/java/io/vertx/ext/web/handler/graphql/impl/GraphQLHandlerImpl.java

```diff
@@ -198,6 +207,67 @@ private void handlePostQuery(RoutingContext rc, GraphQLQuery query, String opera
     executeOne(rc, query);
   }
 
+  private void handlePostMultipart(RoutingContext rc, String operationName, Map<String, Object> variables) {
+    GraphQLInput graphQLInput;
+    if (!options.isRequestMultipartEnabled()) {
+      rc.fail(415);
+      return;
+    }
+
+    try {
+      graphQLInput = parseApolloAttributes(rc);
+    } catch (Exception e) {
+      rc.fail(400, e);
+      return;
+    }
+
+    handlePostQuery(rc, (GraphQLQuery) graphQLInput, operationName, variables);
+  }
+
+  private GraphQLInput parseApolloAttributes(RoutingContext rc) {
+    final MultiMap attrs = rc.request().formAttributes();
+    final Map<String, List<String>> map = Json.decodeValue(attrs.get("map"),
+        new TypeReference<HashMap<String, List<String>>>() {});
+    final GraphQLQuery input = Json.decodeValue(attrs.get("operations"), GraphQLQuery.class);
```

Comment:
According to the spec, the `operations` attribute can be a single request (object) or batch (array)